src/pages/sicp/Sicp.tsx

```diff
@@ -89,13 +94,31 @@ const Sicp: React.FC = () => {
 
   // Handle loading of latest viewed section and fetch json data
   React.useEffect(() => {
+    const valid_langs = ['en', 'zh_CN'];
```

Comment:
1. Use camelCase for variables
2. This is a constant, can we pull it out of the function into a `AVAILABLE_SICP_LANGUAGES` const?
3. Better to use `as const` directive to avoid the language code being typed as a `string`